integration/simulation/simulation.go

```diff
@@ -16,41 +16,57 @@ const (
 	INITIAL_BALANCE = 5000 // nolint:revive,stylecheck
 )
 
-// Simulation represents the data which to set up and run a simulated network
+// SimParams are the parameters for setting up the simulation.
+type SimParams struct {
+	NumberOfNodes   int
+	NumberOfWallets int
+
+	// A critical parameter of the simulation. The value should be as low as possible, as long as the test is still meaningful
+	AvgBlockDurationUSecs uint64
+	AvgNetworkLatency     uint64 // artificial latency injected between sending and receiving messages on the mock network
+	AvgGossipPeriod       uint64 // POBI protocol setting
+
+	SimulationTimeSecs  int // in seconds
+	SimulationTimeUSecs int // SimulationTimeSecs converted to Us
```

Comment:
Feels like these two should just be one 🤔

Reply:
I can tackle this at a later PR :)

Reply:
pls add this to the next pr. I agree. Or maybe we could make a method for conversion.